src/chapters/sql-part2/exercises.rst

```diff
@@ -0,0 +1,97 @@
+Exercises: SQL, Part 2
+=======================
+
+In the previous chapter, we set up a database to store information about seeds. Let's expand that idea now that we know all about foreign keys!
```

Comment:
How about adding an internal link here so that clicking on "previous chapter" sends the student to the SQL part 1 TOC or the relevant ``seeds`` page?

Reply:
Done! Thanks!